docs/_index.md

```diff
@@ -29,7 +44,7 @@ packaged up in the easy-to-understand
 [Super JSON data format](formats/jsup.md) and
 [SuperSQL language](language/_index.md).
 
-While `super` and its accompanying data formats are production quality, the project's
+While `super` and its accompanying data formats are production quality for some use cases, the project's
```

Comment:
I figured saying a flat "production quality" clashed with our message of "expect speedbumps". Those of us who work on it know that years of real world usage of `zq` has made `super` in its current form quite solid for many production use cases, e.g., things for which one might otherwise reach for `jq`. But that history will not be obvious to a new SQL-centric user, so I figured we could just tone down the language a tad.